### PR TITLE
Update master README.md

### PR DESCRIPTION
**Pull request by sharifsalah**
Updates the master README as follows:

Fixes typos
Proof reading, including grammar
New titles
More consistent terminology (project vs sample)
Adjust formatting
Updates contributors
Various style changes

### CODE REVIEW

**Comment by florina-muntenescu** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -2,94 +2,99 @@
...
 
-You can use these samples as a reference or as a starting point for creating
-your own apps. The focus here is on code structure, architecture, testing and
-maintainability. However, bear in mind that there are many ways to build apps
-with these architectures and tools, depending on your priorities, so these
-shouldn't be considered canonical examples. The UI is deliberately kept simple.
+You can use the samples in this project as a learning reference, or as a starting point for creating your own apps. The focus of this project is on demonstrating how to structure your code, design your architecture, and the eventual impact of adopting these patterns on testing and maintaining your app. You can use the techniques demonstrated here in many different ways to build apps. Your own particular priorities will impact how you implement the concepts in these projects, so you should not consider these samples to be canonical examples. To ensure the focus in kept on the aims described above, the app uses a simple UI.
```
Comment (JoseAlcerreca) [inline README.md]: focus is kept

**Comment by JoseAlcerreca** [comment]
LGTM

File: README.md (hunk 1 above)
Comment (sharifsalah) [inline README.md]: The phrase 'is kept' is [passive](https://en.wikipedia.org/wiki/Active_voice), so I suggest keeping it more focused as it is.

**Pull request merged**